Issue when publishing to registry

Hi, I'm getting this error when trying to publish a component to registry

```

> iltorb@1.3.5 install /builds/alvarium/services/holidays/widgets/node_modules/iltorb
> node-pre-gyp install --fallback-to-build

module.js:471
    throw err;
    ^

Error: Cannot find module 'buffer-shims'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/builds/alvarium/services/holidays/widgets/node_modules/iltorb/node_modules/readable-stream/lib/_stream_readable.js:33:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
npm info lifecycle iltorb@1.3.5~install: Failed to exec install script
Error: iltorb@1.3.5 install: `node-pre-gyp install --fallback-to-build`
Exit status 1

/usr/local/lib/node_modules/oc/src/cli/domain/load-dependencies.js:41
      throw err;
      ^
Error: iltorb@1.3.5 install: `node-pre-gyp install --fallback-to-build`
Exit status 1
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/oc/node_modules/npm/lib/utils/lifecycle.js:283:16)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/oc/node_modules/npm/lib/utils/spawn.js:40:14)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:891:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
ERROR: Job failed: exit code 1
```
It happens when oc is trying to load dependencies: `Ensuring dependencies are loaded...FAIL`

Registry version: `0.40.1`
oc-client version: `0.40.1`

`.gitlab-ci.yml` related script:

```yaml
deploy:
  stage: deploy
  image: ciricihq/node:lts-oc
  only:
  - tags
  script:
  - yarn --only=production
  - yarn run build
  - oc registry add https://our-registry-url.com
  - NODE_ENV=production oc publish .
```

Any thoughts about this?

--- a/package.json
+++ b/package.json
@@ -71,7 +71,7 @@
     "multer": "^1.3.0",
     "nice-cache": "0.0.5",
     "node-dir": "0.1.16",
-    "npm": "5.0.2",
+    "npm": "5.3.0",
     "oc-client-browser": "1.0.1",
     "oc-client": "2.1.21",
     "oc-template-handlebars": "6.0.2",

--- a/src/cli/domain/get-missing-deps.js
+++ b/src/cli/domain/get-missing-deps.js
@@ -21,7 +21,7 @@ module.exports = dependencies => {
         delete require.cache[pathToModule];
       }
 
-      require(pathToModule);
+      require.resolve(pathToModule);
     } catch (e) {
       missing.push(npmModule);
     }

